apps/commerce-api/src/main/kotlin/com/loopers/application/order/OrderFacade.kt

```
@@ -0,0 +1,38 @@
+    private val orderService: OrderService,
+    private val pointService: PointService,
+    private val productService: ProductService,
+) {
+
+    @Transactional
+    fun order(userId: Long, command: OrderCommand): OrderModel {
+        val totalPrice = command.orderItems.sumOf { it.productPrice }
+
+        try {
+            pointService.pay(userId, totalPrice)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)